morpcc/navigator.py

```diff
@@ -65,14 +65,14 @@ def __init__(self, schema, entity, request):
 
     def add_attribute(
         self,
-        name,
-        type_,
-        title,
-        description=None,
-        required=False,
-        primary_key=False,
+        name: str,
+        type_: typing.Union[int, str, float],
```

Comment:
u sure this accept int, str, float? .. read create_app.py .. you should see that appadd_attribute pass string there. 
```suggestion
        type_: str,
```